thorgame/ro315/Modified code after review.py

```diff
@@ -0,0 +1,85 @@
+# ## Power of Thor - Episode 1
+#
+# **Link to the problem :
+# [CodinGame Power of Thor](https://www.codingame.com/training/easy/power-of-thor-episode-1)**
+#
+# ### **Problem Overview:**
+#
+# <span class="hljs-comment">In this puzzle, Thor needs to move toward the
```

Comment:
Remove hilights from the reviewer.

Reply:
The highlights from the reviewer weren't in the revised code.